packages/remix/src/generators/error-boundary/schema.d.ts

```diff
@@ -0,0 +1,6 @@
+export interface ErrorBoundarySchema {
+  project: string;
+  routePath: string;
```

Comment:
In the other generators, this variable is named `path`. We should keep it consistent. If we want to change the name across all generators, now would be the time to do it.

Reply:
sure thing! we should stick to `path`, i just wanted to give more context to what `path` was